cypress/integration/specialFeatures/scriptSwitching/testsForAMPOnly.js

```diff
@@ -0,0 +1,67 @@
+import visitPage from '../../../support/helpers/visitPage';
+import { nonCookieVariantAssertions } from '../utilities/scriptSwitchingJourneyAssertions';
+import {
+  clickScriptSwitcher,
+  clickHomePageLink,
+  clickPromoLinkOnHomePage,
+} from '../utilities/scriptSwitchingJourneyActions';
+import {
+  getPrivacyBannerAccept,
+  getCookieBannerAccept,
+} from '../utilities/cookiePrivacyBanner';
+
+export default ({
+  serviceId,
+  serviceName,
+  pageType,
+  path,
+  variant,
+  otherVariant,
+}) => {
+  describe(`Script Switching - ${serviceId} - ${pageType} - ${path}`, () => {
+    beforeEach(() => {
+      cy.clearCookies();
+      visitPage(path, pageType);
+    });
+
+    it(`should change to the correct script when switching script between ${variant} and ${otherVariant}`, () => {
+      // Accept privacy banner
+      getPrivacyBannerAccept(serviceId, variant).click();
+
+      // Accept cookie banner
+      getCookieBannerAccept(serviceId, variant).click();
+
+      cy.log('1st assertion - variant');
```

Comment:
Would it make sense to put the contents of the comment into the cy.log, including the value of the variant?

e.g. 
```
cy.log(`1. Asserting script switch button, url and document lang against variant: ${variant}`);`
```

Reply:
Ah nice idea. These logs were just for debugging, but I like incorporating the comments into them come release.